### PR TITLE
Removed VC activities that didn't work

### LINKED ISSUE #362: Not all of the VC activities work
**Describe the bug**
When trying to play certain activities, an error will pop up saying how the server isn't whitelisted.

**Expected behavior**
The activity to start

**To Reproduce**
1. Run the vc activity command
2. Press the link to start the activity
3. See error

If your activity works, try another one.

**Screenshots**
image

# How to fix

A fix is rather simple.

You'll have to try each activity, if one doesn't work due to the error seen above, you write that activity down.
After you went through all activities, remove the ones that don't work.

### PR DESCRIPTION
**Pull request by Tais993**
See the issue linked below for more info

Closes #362

### CODE REVIEW

File: application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java [hunk 1]
```diff
@@ -227,4 +219,4 @@ private static void handleErrors(@NotNull SlashCommandEvent event,
         return option == null ? null : Math.toIntExact(option.getAsLong());
 
     }
-}
+}
```
Comment (marko-radosavljevic) [inline application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java:222]: no newline :disappointed:

**Review by marko-radosavljevic — APPROVED** [review]

File: application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java (hunk 1 above)
Comment (Tais993) [reply application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java:222]: @Zabuzard gimme tutorial on how to add it as automatic action when commit, too lazy

Comment (Tais993) [reply application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java:222]:
```suggestion
}

```

**Review by marko-radosavljevic — DISMISSED** [review]

**Review by Zabuzard — APPROVED** [review]

**Review by marko-radosavljevic — APPROVED** [review]

**Comment by marko-radosavljevic** [comment]
How did it pass spotless task before, if it was incorrect? If it was correct, why does it pass now? :(

**Comment by Tais993** [comment]
Spotless is asleep now, I fixed it :)

**Review by Zabuzard — APPROVED** [review]

**Pull request merged**